src/vault.ts

```diff
@@ -36,12 +59,7 @@ export default class Vault<StorableItem extends Dictionary> {
     public storeItems(items: StorableItem[]): void {
         this.contents = this.getItems();
```

Comment:
I think line 60 and 64 can be removed since this now invokes `storeItem` which `savesItem` in that function